definitions/region/connections.yaml

```
@@ -0,0 +1,193 @@
+- Connections:
+# allow directional connections explicitly
+- Austria>Czechia
```

The (directional) region names following the hierarchy have to be indented, like in the example shown in the other thread.
```suggestion
    - Austria>Czechia
```